xml/admin_integration.xml

```diff
@@ -30,14 +30,14 @@
   <itemizedlist>
    <listitem>
     <para>
-     &productname; can access the SES cluster.
+     The &productname; cluster can access the SES cluster.
     </para>
    </listitem>
    <listitem>
     <para>
-     &productname; can communicate with the SES nodes&mdash;master, monitoring
-     nodes, OSD nodes and the meta data server in case you need a shared
-     file system. For more details regarding SES refer to
+     That &productname; can communicate with the SES nodes&mdash;master,
+     monitoring nodes, OSD nodes and the metadata server, in case you need a
+     shared file system. For more details regarding SES refer to
      <link xlink:href="https://www.suse.com/documentation/ses-4/book_storage_admin/data/book_storage_admin.html">SES
```

Comment:
I think we need an article here "the SES documentation", not just "SES documentation". Please correct me if I'm wrong.

Shouldn't we use an entity for SES anyway? Just saying, that might be better taken care off in a different PR.

Reply:
Agreed. Fixed -- I have also moved the link text outside the tag, which is more consistent with the other Guides.

I did bring entities for SES, MicroOS and a few other things across from the release-notes entities file.